public/index.html

```diff
@@ -21,6 +21,13 @@
         <div id = "app">
         </div>
         
+        <script type="text/jsx">
```

Comment:
This is unnecessary because `webpack` automatically compiles `.js` files written and bundles them into `/dist/build/bundle.js`